logictest/schemas/default.json

```diff
@@ -0,0 +1,60 @@
+{
+    "name": "default",
```

Comment:
Love this idea. Is there a reason to add an additional json parsing layer? In my mind we'd do something like:
```
var schemas = map[string]*dynparquet.Schema{
    "default": dynparquet.NewSampleSchema(),
    "bools": dynparquet.SchemaFromDefinition(
        < your bools def here >
    )
}
```

I'm open to this json idea, just curious.

Reply:
I typically don't put configuration settings into code, but I don't have strong feelings against doing it that way